classes/migration/install/CommonMigration.php

```diff
@@ -45,7 +45,7 @@ public function up(): void
             $table->bigInteger('redirect')->default(0)->comment('If not 0, redirect to the specified journal/conference/... site.');
             $table->string('primary_locale', 14)->comment('Primary locale for the site.');
             $table->smallInteger('min_password_length')->default(6);
-            $table->string('installed_locales', 1024)->default('en_US')->comment('Locales for which support has been installed.');
+            $table->string('installed_locales', 1024)->default('en')->comment('Locales for which support has been installed.');
```

Comment:
You'll also need to change the default for existing schema in the upgrade migration (here for `site` and below for `email_templates_default_data`). At a glance, https://stackoverflow.com/questions/37003469/laravel-migrations-change-default-value-of-column has a solution.